### DISCUSSION TITLE
No .exe file in the zip?

### DISCUSSION THREAD

**Discussion by lbrisbois8** [chummer5a/chummer5a#5003] [Q&A]
This seems like a really silly issue, but I downloaded the most recent update and it crashed my chummer so it wouldn't open. I decided to just reinstall the program cause this has happened before and troubleshooting it just ended up running me in circles.

When I installed it, I extracted the files but can't find the .exe file to open the application. I can't tell if I've just forgotten where it normally puts the .exe or if it's missing from the zip.

**Comment by chummer5a** [comment]
You've downloaded the source files, not the release. Go to https://github.com/chummer5a/chummer5a/releases or https://github.com/chummer5a/chummer5a/releases/latest and expand Assets.
